README.md

```diff
@@ -153,16 +177,36 @@ Following environment variables should be set:
 AWS_LAMBDA_EXEC_WRAPPER: /opt/otel-instrument
 SPLUNK_ACCESS_TOKEN: <org_access_token>
 SPLUNK_REALM: us0
+OTEL_EXPORTER_OTLP_TRACES_PROTOCOL: http/protobuf
+```
+
+#### Node.js
+
+The example assumes the following:
+
+- Data is sent to Splunk APM directly via Ingest endpoint.
+- Context propagation is default (`tracecontext,baggage`).
+- Realm is `us0`.
+
+Following environment variables should be set:
+```
+AWS_LAMBDA_EXEC_WRAPPER: /opt/nodejs-otel-handler
+SPLUNK_ACCESS_TOKEN: <org_access_token>
+SPLUNK_REALM: us0
+OTEL_TRACES_EXPORTER: jaeger-thrift-splunk
 ```
 
 ## Troubleshooting
 
 ### Java
 
-1. If traces are not delivered 
+1. If traces are not showing in APM 
+    - Set `OTEL_LOG_LEVEL` to `DEBUG` and check the CloudWatch logs.
     - Set `OTEL_LAMBDA_LOG_LEVEL` to `DEBUG` and search for traces IDs in the backend.
-    - Set `OTEL_LAMBDA_LOG_LEVEL` to `DEBUG` for the `jaeger-thrift-splunk` exporter. This causes exporter to log debug information.
-    - Increase `OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT` if the backend / network reacts slowly.
+    - Set `OTEL_LAMBDA_LOG_LEVEL` to `DEBUG` for the `jaeger-thrift-splunk' exporter. This shows debug information for the exporter.
```

Comment:
How's that different from the previous step?

Reply:
With j/t exporter you will see additional info - ie actual calls to the backend. So it's about the combination rather than a single value.